source/includes/steps-install-shell-linux-deb.yaml

```diff
@@ -0,0 +1,26 @@
+---
+source:
+  file: steps-install-shell-base.yaml
+  ref: navigate-dlc
+ref: navigate-dlc-linux-deb
+---
+title: "Download the Debian 64-bit ``.deb`` package."
+ref: download-archive-rhel
+level: 4 
+---
+title: "Extract and install the ``.deb`` package."
+ref: extract-archive-linux-deb
+level: 4
+content: |
+  To install the downloaded ``.deb`` package, either:
+
+  - Double click your downloaded ``.deb`` package icon and follow
+    the prompts.
```

Comment:
• This is a bit strange, as even DEB users who do have GUI / desktops would still likely run the command line step for this install. You can do the same thing for RPMs on RHEL (say on CentOS with GNOME), but that step is not present in the RPM steps. I think I'd just remove the GUI step, simplify this step a bit.